syntax = "proto3";

package MyRPC;

message RpcHeader 
{
    bytes serviceName = 1; 
    bytes methodName = 2;
    uint32 argvSize = 3;    
}